hack/tools/.custom-gcl.yaml

```diff
@@ -0,0 +1,6 @@
+version:  v1.63.4
+name: golangci-lint-kal-v1.63.4
+destination: ./bin
+plugins:
+- module: 'github.com/JoelSpeed/kal'
```

Comment:
If I'm understanding correctly, if/when the project's accepted in kubernetes-sigs, this file is the only thing that needs to change, correct?

Reply:
Yep, we would need to update this to point to the new source. There may of course be other changes over time since this project is early in development, but, I would hope that we can keep this fairly compatible with what we are doing here and now

My aim at least is that it stays as a golangci-lint plugin until and unless it replaces the flexibility of the configuration already implemented by golangci-lint. But there's an awful lot of functionality there and I don't see much value in re-implementing that.